src/test/java/teammates/test/pageobjects/StudentProfilePage.java

```diff
@@ -150,7 +150,7 @@ public void selectGender(String gender) {
         }
     }
 
-    public void editProfileThroughUi(String fileName, String shortName, String email, String institute,
+    public void editProfileThroughUi(String shortName, String email, String institute,
```

Comment:
When you change the source file corresponding test files also will be affected. You need to change the test file also. That's why the build is failing.